department-of-veterans-affairs/vets-website
# 85115 1010 ezr update submit error message

## Summary

- Updates the content for the submission error message, which includes a link to the static page.

## Related issue(s)

- https://app.zenhub.com/workspaces/10-10-health-apps-5fff0cfd1462b6000e320fc7/issues/gh/department-of-veterans-affairs/va.gov-team/85115

## Testing done

- This can be done in a review instance or staging: 
1. Log in as a staging user and fill out the EZR form. 
2. Once you reach the review page, right click in your browser window, select inspect, go to the "Application" tab, click on "Local storage", and click on the first item in the dropdown.
3. You should see a key named `crsfToken`. Double-click on the value and change it to something else.
4. Submit the form and you should see the updated submission error message.

## Screenshots
### Before
![](https://github.com/department-of-veterans-affairs/vets-website/assets/34111449/b63743e6-fdca-47fb-aa75-4c196670618c)
### After
![](https://github.com/department-of-veterans-affairs/vets-website/assets/34111449/c097f820-1a22-4234-895c-cc306b2714fc)
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExNTV2YXE4azAyanVpYms3MTMzZnlnZTUybWJuaWZzamQ5cXJjandqNyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/PEbphaoZsbgjKogDXl/giphy.gif)

## What areas of the site does it impact?

- 10-10EZR form

## Acceptance criteria

### Quality Assurance & Testing


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
